src/yadmark/benchmark/external/apfelpy_utils.py

```diff
@@ -0,0 +1,346 @@
+"""Utilities to help run APFEL++ benchmarks."""
+import numpy as np
+from eko import basis_rotation as br
+
+# Q2 knots specs
+NQ = 250
+QMin = 1
+QMax = 200
+
+# Map observables names to APFEL++ methods
+MAP_HEAVINESS = {
+    "charm": 4,
+    "bottom": 5,
+    "top": 6,
+    "light": 0,  # TODO: Check combination
```

Comment:
what is the status of this todo?

Reply:
They don't match. So either we leave it as pending either we remove

Reply:
Here the best would be to leave it as pending.